### PR TITLE
Model 1 Causes Artifact

### PR DESCRIPTION
**Pull request by hussain-jafari**
### Description
- *Category*: data artifact
- *JIRA issue*: [MIC-2900](https://jira.ihme.washington.edu/browse/MIC-2900)
- *Research reference*: [Cause Models](https://vivarium-research.readthedocs.io/en/latest/concept_models/vivarium_iv_iron/child_model/concept_model.html#cause-models)

### Verification and Testing
Created artifacts for South Asia and Sub-Saharan Africa and checked that data was complete and looked reasonable.

### CODE REVIEW

File: src/vivarium_gates_child_iv_iron/constants/results.py [hunk 1]
```diff
@@ -26,49 +26,58 @@
...
-THROWAWAY_COLUMNS = [f'{state}_event_count' for state in models.STATES]
-
 TOTAL_POPULATION_COLUMN_TEMPLATE = 'total_population_{POP_STATE}'
 PERSON_TIME_COLUMN_TEMPLATE = 'person_time_in_{YEAR}_among_{SEX}_in_age_group_{AGE_GROUP}'
 DEATH_COLUMN_TEMPLATE = 'death_due_to_{CAUSE_OF_DEATH}_in_{YEAR}_among_{SEX}_in_age_group_{AGE_GROUP}'
 YLLS_COLUMN_TEMPLATE = 'ylls_due_to_{CAUSE_OF_DEATH}_in_{YEAR}_among_{SEX}_in_age_group_{AGE_GROUP}'
 YLDS_COLUMN_TEMPLATE = 'ylds_due_to_{CAUSE_OF_DISABILITY}_in_{YEAR}_among_{SEX}_in_age_group_{AGE_GROUP}'
 STATE_PERSON_TIME_COLUMN_TEMPLATE = '{STATE}_person_time_in_{YEAR}_among_{SEX}_in_age_group_{AGE_GROUP}'
 TRANSITION_COUNT_COLUMN_TEMPLATE = '{TRANSITION}_event_count_in_{YEAR}_among_{SEX}_in_age_group_{AGE_GROUP}'
+DISEASE_STATE_PERSON_TIME_COLUMN_TEMPLATE = '{DISEASE_STATE_EXCL_DIARRHEA}_person_time_in_{YEAR}_among_{SEX}_in_age_group_{AGE_GROUP}_wasting_state_{WASTING_STATE}'
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/constants/results.py:36]: This isn't actually related to artifact building, so it should have been in a separate PR (one for the post processing ticket). Each of these column templates has several stratifications that haven't been added to the model yet (like wasting) and others that will never be added to the model (like therapeutic zinc).

File: src/vivarium_gates_child_iv_iron/constants/models.py [hunk 2]
```diff
@@ -10,29 +12,31 @@ def __new__(cls, value):
         return obj
 
 
+# noinspection PyPep8Naming
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/constants/models.py:15]: This stuff should be added in a PR for the model implementation ticket, but it's fine for now. Just something to bear in mind for future model versions.

File: src/vivarium_gates_child_iv_iron/model_specifications/iv_iron_child.yaml [hunk 3]
```diff
@@ -5,6 +5,10 @@ components:
             - BasePopulation()
             - Mortality()
             - FertilityCrudeBirthRate()
+    vivarium_gates_child_iv_iron:
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/model_specifications/iv_iron_child.yaml]: This should be added in the Observers PR.

File: src/vivarium_gates_child_iv_iron/tools/make_artifacts.py [hunk 4]
```diff
@@ -97,7 +97,7 @@ def build_artifacts(
     check_for_existing(Path(output_dir), location, append, replace_keys)
 
     if location in metadata.LOCATIONS:
-            build_single(location, output_dir, replace_keys)
+        build_single(location, output_dir, replace_keys)
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/tools/make_artifacts.py:100]: Is this an issue in the research template? If so, can you add it to your open research template PR

File: src/vivarium_gates_child_iv_iron/utilities.py [hunk 5]
```diff
@@ -128,6 +128,29 @@ def _get_standard_deviation(
     return sd
 
 
+def get_norm_from_quantiles(mean: float, lower: float, upper: float,
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/utilities.py]: Don't use these functions. You want to use the existing functions above `get_norm` and `get_truncnorm`.

File: src/vivarium_gates_child_iv_iron/constants/data_values.py [hunk 6]
```diff
@@ -1,4 +1,37 @@
...
+    get_truncnorm_from_sd
+)
+
+##########################
+# Cause Model Parameters #
+##########################
+
+# diarrhea duration in days
+DIARRHEA_DURATION: Tuple = (
+    'diarrheal_diseases_duration', get_norm_from_quantiles(mean=4.04485, lower=3.94472, upper=4.144975)
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/constants/data_values.py]: See comment in `utilities.py`

**Review by rmudambi — CHANGES_REQUESTED** [review]

**Comment by mattkappel** [comment]
The Jira ticket is MIC-2900

**Review by rmudambi — APPROVED** [review]

**Review by mattkappel — APPROVED** [review]

**Pull request merged**